### PR TITLE
Feature/Cc-160

### PR DESCRIPTION
**Pull request by suNrisEinMyeYes**

### CODE REVIEW

File: contracts/marketplace/CruzoMarket.sol [hunk 1]
```diff
@@ -18,6 +28,90 @@ contract CruzoMarket is
...
+        if (_value != tradePrice * _amount) {
+            revert Market__IncorrectSentEtherValue(
+                _value,
+                tradePrice * _amount
+            );
+        }
+        _;
+    }
+
+    modifier isReceiveeCorrect(address _receiver) {
```
Comment (bobrom) [inline contracts/marketplace/CruzoMarket.sol]: @suNrisEinMyeYes could you please adjust modifier name to **isReceiverCorrect** for the consistency purpose

File: hardhat.config.ts [hunk 2]
```diff
@@ -154,38 +155,38 @@ const config: HardhatUserConfig = {
       // gasPrice: 2000000000,
     },
   },
-  etherscan: {
```
Comment (bobrom) [inline hardhat.config.ts]: @suNrisEinMyeYes Why etherscan configuration is commented? How to verify contracts?

File: scripts/deploy/deployVault.ts [hunk 3]
```diff
@@ -8,7 +8,11 @@ async function main() {
     }
 
     console.log("Deploying vault contract");
+<<<<<<< HEAD
```
Comment (bobrom) [inline scripts/deploy/deployVault.ts]: @suNrisEinMyeYes What's this?

**Review by bobrom — CHANGES_REQUESTED** [review]

File: hardhat.config.ts (hunk 2 above)
Comment (suNrisEinMyeYes) [reply hardhat.config.ts]: image

Comment (suNrisEinMyeYes) [reply hardhat.config.ts]: I didn't implement this part, so I don't know why it is marked as error

File: scripts/upgrade/upgradeVault.ts [hunk 4]
```diff
@@ -16,7 +16,11 @@ async function main() {
     }
 
     console.log("Upgrading vault contract");
+<<<<<<< HEAD
```
Comment (bobrom) [inline scripts/upgrade/upgradeVault.ts]: Something from merge?

File: hardhat.config.ts (hunk 2 above)
Comment (bobrom) [reply hardhat.config.ts]: Works fine on my side. Did you try to install latest dependencies `yarn install`?

**Review by bobrom — APPROVED** [review]

**Pull request merged**